src/App.tsx

```diff
@@ -1,61 +1,40 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { Form } from './components/Form/Form';
+import { getUserById } from './services/user';
+import { EnrichedTodo } from './types/EnrichedTodo';
+import { useState } from 'react';
+
+export const userTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
```

Comment:
Ensure that the `getUserById` function returns a valid user object or `null`. The `TodoList` component should handle `null` values appropriately to avoid rendering issues.